dokka-subprojects/analysis-kotlin-symbols/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/symbols/plugin/KotlinAnalysis.kt

```diff
@@ -79,6 +80,8 @@ internal fun createAnalysisSession(
     projectDisposable: Disposable = Disposer.newDisposable("StandaloneAnalysisAPISession.project"),
     isSampleProject: Boolean = false
 ): KotlinAnalysis {
+    enableExperimentalKDocResolution()
```

Comment:
Can we make this under a flag in Dokka untill `experimentalKDocResolution` is enabled by default?